examples/anthos-service-mesh-multicluster/infrastructure/terraform.tfvars

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+project_id="your-project-id" # Replace with your project ID
```

Comment:
If a user runs this config is as is without reading instructions, Terraform will not complain about a missing project ID due to these tfvars. I would recommend renaming this as `terraform.example.tfvars` and then explicitly instructing to rename this and fill in vars.